lib/bundler/source.rb

```diff
@@ -13,7 +13,7 @@ def unmet_deps
     def version_message(spec)
       message = "#{spec.name} #{spec.version}"
 
-      if Bundler.locked_gems
+      if !Bundler.plugin_install_mode && Bundler.locked_gems
```

Comment:
While installing a plugin in a directory without Gemfile, 

like so
`bundle plugin install whatever`

when I use the installer, to install the plugins, the installer checks for the locked gems to show "(locked at whatever)" and crashes.

So, I created a "plugin_install_mode" where it won't worry about Gemfile and Lockfile. It doesn't seem all that elegant, but works.